### PR TITLE
Fix loading screen not showing when opening a budget file

### PR DESCRIPTION
**Pull request by joel-jeremy**

### CODE REVIEW

File: packages/desktop-client/src/components/App.tsx [hunk 1]
```diff
@@ -38,37 +44,49 @@ type AppInnerProps = {
 function AppInner({ budgetId, cloudFileId }: AppInnerProps) {
   const [initializing, setInitializing] = useState(true);
   const { showBoundary: showErrorBoundary } = useErrorBoundary();
-  const stateLoadingText = useSelector((state: State) => state.app.loadingText);
-  const [loadingText = stateLoadingText, setLoadingText] = useState<
-    string | null
-  >(null);
-  const { loadBudget, closeBudget, loadGlobalPrefs } = useActions();
+  const loadingText = useSelector((state: State) => state.app.loadingText);
+  const dispatch = useDispatch();
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/App.tsx:48]: **👏 praise:** ‏ohhhhh... right, the local state I implemented does not listen to `state.app.loadingText` updates. 🤦 
Thanks for patching this!

File: packages/desktop-client/src/components/AppBackground.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function AppBackground({
   const transitions = useTransition(loadingText, {
     from: { opacity: 0, transform: 'translateY(-100px)' },
     enter: { opacity: 1, transform: 'translateY(0)' },
-    leave: { opacity: 0, transform: 'translateY(-100px)' },
+    leave: { opacity: 0, transform: 'translateY(100)' },
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/AppBackground.tsx:25]: **❓ question:** ‏did you mean to change this to `100px`? Why do this?
```suggestion
    leave: { opacity: 0, transform: 'translateY(100px)' },
```

File: packages/desktop-client/src/components/manager/BudgetList.jsx [hunk 3]
```diff
@@ -294,47 +298,51 @@ export function BudgetList() {
...
           Import file
         </Button>
 
-        <Button type="primary" onClick={onCreate} style={{ marginLeft: 15 }}>
+        <Button
+          type="primary"
+          onClick={onCreate}
+          style={{
+            ...narrowButtonStyle,
+          }}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/BudgetList.jsx:344]: **🥜 nitpick:** ‏
```suggestion
          style={narrowButtonStyle}
```

File: packages/desktop-client/src/components/manager/BudgetList.jsx [hunk 4]
```diff
@@ -343,7 +351,9 @@ export function BudgetList() {
             type="primary"
             isSubmit={false}
             onClick={() => onCreate({ testMode: true })}
-            style={{ marginLeft: 15 }}
+            style={{
+              ...narrowButtonStyle,
+            }}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/BudgetList.jsx:356]: **🥜 nitpick:** ‏
```suggestion
            style={narrowButtonStyle}
```

**Review by MatissJanis — COMMENTED** [review]
Thanks!

File: packages/desktop-client/src/components/AppBackground.tsx (hunk 2 above)
Comment (joel-jeremy) [reply packages/desktop-client/src/components/AppBackground.tsx]: Yes, it is smoother IMO, it prevents overlapping of the next loading message with the previous one.

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**